database/databaseinterface.go

```diff
@@ -0,0 +1,24 @@
+package database
+
+import (
+	obj "github.com/jcgallegdup/Concurrent-Document-Editor/objects"
+)
+
+// Database contains all the functions that interact with the database.
+type DatabaseInterface interface {
+	// Creates a new file, stores it in the database, and returns the id for it.
+	CreateEmptyFile(fileName string, userID string) (int, error)

```

Comment:
The function `createEmptyFile()` needs to be changed to return a file ID, revision number, and an error in order to match up with what is in master right now and resolve merge conflicts